plugins/woocommerce/src/Admin/API/Patterns.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Admin\API;
+
+use Automattic\WooCommerce\Blocks\BlockPatterns;
+use Automattic\WooCommerce\Blocks\Package;
+use Automattic\WooCommerce\Blocks\Patterns\PTKClient;
+use Automattic\WooCommerce\Blocks\Patterns\PTKPatternsStore;
+use Automattic\WooCommerce\StoreApi\Exceptions\RouteException;
+use Exception;
+use WP_Error;
+use WP_HTTP_Response;
+use WP_REST_Response;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Product controller
+ *
+ * @internal
+ */
+class Patterns {
+
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc-admin';
```

Comment:
If we're putting `$namespace` in a variable here shall we do the same with `/patterns` in `$endpoint`?

Reply:
I don't think it's necessary to have this variable 👍